tensorflow/contrib/cmake/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2.0
+
+package(default_visibility = ["//visibility:public"])
+
+load("//tensorflow:tensorflow.bzl", "py_test")
+
+py_test(
+    name = "python_modules_test",
+    size = "small",
+    srcs = ["python_modules_test.py"],
+    data = [
+        "//tensorflow:all_contrib_files",
+    ],
```

Comment:
You also need to add python_modules.txt as a data dependency here.

Reply:
Had that here, but is already inside `contrib`.

Reply:
Also, `pip_smoke_test.py` complains about them not being in the PIP package.
Shall I whitelist the `.txt` files or the `_test` targets ?